deployment/openshift/bpm_dc.yaml

```diff
@@ -418,7 +418,7 @@ parameters:
   - name: CAMUNDA_JOB_MAX_WAIT
     description: CAMUNDA_JOB_MAX_WAIT
     required: true
-    value: "60000"
+    value: "300000"
```

Comment:
Iman can you review this specifc value, Seems we don't want to increase waiting time from 1min to 5min. If this cause any and increasing time required then continue with 5min for now.